test-configs.yaml

```diff
@@ -485,11 +485,41 @@ device_types:
     filters:
       - blacklist: *allmodconfig_filter
 
+  fsl-ls1012a-rdb:
+    mach: freescale
+    class: arm64-dtb
+    boot_method: uboot
```

Comment:
Baseline results on staging: https://staging.kernelci.org/test/plan/id/5e9f96b94c8074c7a661e06e/
Kernel panic but no LAVA infrastructure errors so that's fine as far as this PR is concerned (also seems to have been fixed with alternative network card as per the other comments).

Upstream in LAVA:
https://git.lavasoftware.org/lava/lava/-/blob/master/etc/dispatcher-config/device-types/fsl-ls1012a-rdb.jinja2